apps/omg/lib/omg/crypto.ex

```diff
@@ -37,7 +37,7 @@ defmodule OMG.Crypto do
   @doc """
   Recovers address of signer from binary-encoded signature.
   """
-  @spec recover_address(hash_t(), sig_t()) :: {:ok, address_t()} | {:error, :signature_corrupt}
+  @spec recover_address(hash_t(), sig_t()) :: {:ok, address_t()} | {:error, :signature_corrupt | binary}
```

Comment:
FYI, dialyzier is unable to be specific when it comes to binary types. In this case, when we go down the rabbit hole of `:libsecp256k1.ecdsa_recover_compact/4` the error return tuples resolve to: `"Compression flag invalid"`, `"Recovery id invalid 0-3"`, `"ecdsa_signature_parse_compact returned 0"`, `"ecdsa recovery problem"` or `"ecdsa pubkey serialize error"`. Thats the reason why we generally prefer `atom` only tuples.